lib/ask/ecto_types/stats.ex

```diff
@@ -30,7 +32,8 @@ defmodule Ask.Stats do
        total_call_time: total_call_time,
        total_call_time_seconds: total_call_time_seconds,
        call_durations: call_durations || %{},
-       attempts: attempts
+       attempts: attempts,
+       last_call_started: last_call_started || false
```

Comment:
I think that if last_call_started is null it should be translated to `true` in order to keep the same reports for existing data.

Reply:
Handle this in data migration instead of code if possible.